pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -423,5 +428,13 @@ public long getBytesOutCounter() {
         return getStats(false).bytesOutCounter;
     }
 
+    public boolean isBrokerDeleteInactiveTopicsEnabled() {
+        return brokerDeleteInactiveTopicsEnabled;
+    }
+
+    public void setBrokerDeleteInactiveTopicsEnabled(boolean brokerDeleteInactiveTopicsEnabled) {
```

Comment:
How is this supposed to be activated? 

Instead, we need a namespace policy, attached to https://pulsar.apache.org/admin-rest-api/?version=2.5.2#operation/setAutoTopicCreation that adds the auto-deletion override configuration, on top of the auto-creation

Reply:
@merlimat This is used by the protocol handlers who want to ensure some topics under the namespace does not delete by the broker. But do not want to change the namespace level policy. Now, we don't have a topic level policy, without this change, we must create a virtual producer or consume.